### PR TITLE
Feature/Events scene

### PR DESCRIPTION
**Pull request by darek607**
Simple scene that tests all events of UiNode. I used a TextEdit, because it also supports `onFocusGained` and `onFocusLost` events on mobile.

events_scene

### CODE REVIEW

File: src/test_scenes/sceneEvents.js [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+  }
+
+  renderCheckbox(text, on) {
+      return (
+        <Toggle height={0.04} text={text} type="checkbox" on={on} enabled={false}/>
+      )
+  }
+
+  onEnableToggleChanged = event => {
+    this.setState({enabled: !this.state.enabled});
```
Comment (pleszkiewicz) [inline src/test_scenes/sceneEvents.js]: The proper way of setting a new state based on the previous state is as follows:
```
this.setState(state => ({ enabled: !state.enabled }));
```

File: src/test_scenes/sceneEvents.js [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+
+  renderNode() {
+     return ( 
+      <TextEdit
+          width={0.5} 
+          height={0.2} 
+          multiline={true}
+          text={"Interactive node"}
+          enabled={this.state.enabled}
+          onActivate={()=>{this.setState({onActivate: true})}}
```
Comment (pleszkiewicz) [inline src/test_scenes/sceneEvents.js]: I think this:
```
onActivate={()=>{this.setState({onActivate: true})}}
```
could be simplified to the following form:
```
onActivate={() => this.setState({onActivate: true})}
```

File: src/test_scenes/sceneEvents.js [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+        <Toggle height={0.04} text={text} type="checkbox" on={on} enabled={false}/>
+      )
+  }
+
+  onEnableToggleChanged = event => {
+    this.setState({enabled: !this.state.enabled});
+  }
+
+  onDeleteToggleChanged = event => {
+    this.setState({deleted: !this.state.deleted});
```
Comment (pleszkiewicz) [inline src/test_scenes/sceneEvents.js]: Should we reset all state's values back to **true** when the delete toggle is set back to **false**?

**Review by pleszkiewicz — APPROVED** [review]
Code: 👍 
Func: I asked @SPaluchiewiczSofomo to test it on iOS.

File: src/test_scenes/sceneEvents.js (hunk 3 above)
Comment (darek607) [reply src/test_scenes/sceneEvents.js]: I think no, because the goal is to have all check boxes marked to have clear info that all events have fired.

**Review by pleszkiewicz — APPROVED** [review]
👍

**Pull request merged**